web/src/lib/components/admin-page/server-stats/server-stats-panel.svelte

```diff
@@ -21,7 +21,8 @@
     return '0'.repeat(zeroLength);
   };
 
-  $: [statsUsage, statsUsageUnit] = getBytesWithUnit(stats.usage, 0);
+  const TiB = 1024 ** 4;
```

Comment:
Maybe this could be a constant in byte-units?

Reply:
Maybe, if I use it somewhere else 😁